src/gui/editors/PianoRoll.cpp

```diff
@@ -4165,6 +4165,13 @@ void PianoRoll::pasteNotes()
 			cur_note.restoreState( list.item( i ).toElement() );
 			cur_note.setPos( cur_note.pos() + Note::quantized( m_timeLine->pos(), quantization() ) );
 
+			// handle case where notes are copied from BB editor
+			if (cur_note.length().getTicks() == -192)
```

Comment:
Just in case [this ever changes](https://github.com/LMMS/lmms/blob/master/src/tracks/Pattern.cpp#L301)...
```suggestion
			if (cur_note.length().getTicks() == -DefaultTicksPerBar)
```